src/components/__tests__/LibrariesPage-test.tsx

```diff
@@ -102,32 +175,57 @@ describe("LibrariesPage", () => {
       spyClear.restore();
     });
 
+    it("shouldn't display QA search results unless in QA mode", async () => {
+      wrapper.setProps({ results: { libraries: [qaLib] } });
+      wrapper.setState({ searchTerm: "QA Library", showAll: false });
+      let results = wrapper.find(".panel-warning");
+      expect(results.length).to.equal(0);
+
+      wrapper.setState({...wrapper.state(), ...{qa: true}});
+
+      results = wrapper.find(".panel-warning");
+      expect(results.length).to.equal(1);
+      expect(results.find(".panel-title").text()).to.equal("QA Library (qa)");
+    });
+
+    it("should let the user check QA if the production search was unsuccessful", async () => {
+      expect(search.callCount).to.equal(0);
+      wrapper.setState({ searchTerm: "QA Library"});
+      wrapper.instance().toggleQA(true);
```

Comment:
`toggleQA` is asynchronous but `await` didn't work for this (and so the `pause` function)?